math/stirling_number_of_the_first_kind/info.toml

```
@@ -0,0 +1,43 @@
+title = 'Stirling Number of the First Kind'
+timelimit = 5.0
```

```suggestion
timelimit = 10.0
```

一応10sにしておきたいです(なんとなくなんですが、Formal Power Series系はかなり実行時間のばらつきが大きい + python使う人にも人気という傾向があるので)